source/lac/sparse_matrix_inst2.cc

```diff
@@ -0,0 +1,18 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2013 - 2014 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+#define SPLIT_INDEX 1
+#include "sparse_matrix.cc"
```

Comment:
This would lead to confusing errors if we decided to put any function into `sparse_matrix.cc`.  Just copy the contents of that file here.